proto/cosmos/slashing/genesis.proto

```diff
@@ -0,0 +1,54 @@
+syntax = "proto3";
+package cosmos.slashing;
+
+option go_package            = "github.com/cosmos/cosmos-sdk/x/slashing/types";
+option (gogoproto.equal_all) = true;
+
+import "gogoproto/gogo.proto";
+import "cosmos/slashing/slashing.proto";
+
+// GenesisState - all slashing state that must be provided at genesis
+message GenesisState {
+  Params params = 1 [
+    (gogoproto.casttype) = "Params",
+    (gogoproto.nullable) = false
+  ];
+
+  repeated ValidatorSigningInfos signing_infos = 2 [
+    (gogoproto.casttype) = "ValidatorSigningInfos",
+    (gogoproto.moretags) = "yaml:\"signing_infos\"",
+    (gogoproto.nullable) = false
+  ];
+
+  repeated ValidatorMissedBlocks missed_blocks = 3 [
+    (gogoproto.casttype) = "ValidatorMissedBlocks",
+    (gogoproto.moretags) = "yaml:\"missed_blocks\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
+// ValidatorSigningInfos 
+message ValidatorSigningInfos {
```

Comment:
SigningInfos maybe better name, as it has signing info of all vals rather than one